Legacy/bonej/src/main/java/org/bonej/plugins/SliceGeometry.java

```diff
@@ -129,6 +129,7 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 	private boolean[] emptySlices;
 	/** List of slice centroids */
 	private double[][] sliceCentroids;
+	private double[] integratedDensity;
```

Comment:
IntelliJ warns `Contents of array 'integratedDensity' are written to, but never read`. Never added to ResultTable?

Reply:
True. Its intended function is handled by cortical area when partial volume compensation is checked. Removed.